### PR TITLE
replicaset details in summary card

### PR DESCRIPTION
**Pull request by madhuv-msft**
for a small screen upto 750px, here is the UI
image

for other screens
image

### CODE REVIEW

**Comment by madhuv-msft** [comment]
i have one more idea on this, so i am closing for the time being.

**Comment by debubarve** [comment]
Thanks Madhu. 
Why there is a horizontal scrolling appearing for the 750 px screen resolution?

**Comment by madhuv-msft** [comment]
> Thanks Madhu.
> Why there is a horizontal scrolling appearing for the 750 px screen resolution?

we want to set a minimum width, so if user make anything below 700px we will show the scroll.

File: src/WebUI/ImageDetails/ImageDetails.tsx [hunk 1]
```diff
@@ -138,12 +135,7 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
                 );
 
             default:
-                const itemToRender = defaultColumnRenderer(value);
-                return (
-                    <div className="text-ellipsis details-card-value-field-size">
-                        {itemToRender}
-                    </div>
-                );
+                return defaultColumnRenderer(value, "details-card-value-field-size");
```
Comment (madhuv-msft) [inline src/WebUI/ImageDetails/ImageDetails.tsx]: image details, ellipsis were not shown when the text is big

File: src/WebUI/Pods/PodOverview.tsx [hunk 2]
```diff
@@ -146,26 +138,21 @@ export class PodOverview extends BaseComponent<IPodOverviewProps> {
...
-                    rel={"noopener noreferrer"}
-                    excludeTabStop
-                    onClick={(e) => {
-                        e.preventDefault();
-                        showImageDetails(imageId);
-                    }}
-                >
-                    {value}
-                </Link>
+                <div className="pod-image-link details-card-value-field-size">
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodOverview.tsx]: could not be able to use keybaord to reach to this link and also the border of the link is not shown when used with keyboard

File: src/WebUI/WorkLoads/WorkloadDetails.tsx [hunk 3]
```diff
@@ -187,6 +185,9 @@ export class WorkloadDetails extends BaseComponent<IWorkloadDetailsProperties, I
                 selectedPod: pods && pods.length > 0 ? pods[0] : null
             });
         }
+
+        this._adjustWorkloadColumnDetailsSize();
```
Comment (madhuv-msft) [inline src/WebUI/WorkLoads/WorkloadDetails.tsx]: logic of width computation

Comment (madhuv-msft) [reply src/WebUI/WorkLoads/WorkloadDetails.tsx]: removed this logic as we are using flex-grow and it is working perfectly alright.

**Pull request merged**